cql2pgjson/src/test/java/org/folio/cql2pgjson/CQL2PgJSONTest.java

```diff
@@ -500,94 +515,96 @@ public void caret(String testcase) {
   })
   public void unicode(String testcase) {
     select(testcase);
+    
+    
     select(testcase.replace("==", "==/ignoreCase/ignoreAccents ")
                    .replace("= ", "= /ignoreCase/ignoreAccents "));
   }
 
   @Test
   @Parameters({
-    "address.city= /respectCase Søvang # Lea Long",
-    "address.city==/respectCase Søvang # Lea Long",
-    "address.city= /respectCase SØvang #",
-    "address.city==/respectCase SØvang #",
-    "address.city= /respectCase Sovang # Lea Long",
-    "address.city==/respectCase Sovang # Lea Long",
-    "address.city= /respectCase SOvang #",
-    "address.city==/respectCase SOvang #",
-    "address.city= /respectCase Sövang # Lea Long",
-    "address.city==/respectCase Sövang # Lea Long",
-    "address.city= /respectCase SÖvang #",
-    "address.city==/respectCase SÖvang #",
+    "address.city=  Søvang # Lea Long",
+    "address.city== Søvang # Lea Long",    
+    "address.city=  Sovang # Lea Long",
+    "address.city== Sovang # Lea Long",
+    "address.city=  Sövang # Lea Long",
+    "address.city== Sövang # Lea Long",
+    "address.city=  SØvang # Lea Long",
+    "address.city== SØvang #",
+    "address.city=  SOvang # Lea Long",
+    "address.city== SOvang #",
+    "address.city=  SÖvang # Lea Long",
+    "address.city== SÖvang #",
```

Comment:
Using = and == must return the same result because the query string matches the complete field.
Please do not change the unit test to match the bug of the implementation. Can you fix the implementation?